packages/expo-module-template/android/build.gradle

```
@@ -47,11 +47,11 @@ uploadArchives {
 }
 
 android {
-  compileSdkVersion 26
+  compileSdkVersion safeExtGet("compileSdkVersion", 29)
 
   defaultConfig {
-    minSdkVersion 21
-    targetSdkVersion 26
+    minSdkVersion safeExtGet("minSdkVersion", 21)
+    targetSdkVersion safeExtGet("targetSdkVersion", 29)
```

```suggestion
    targetSdkVersion safeExtGet("targetSdkVersion", 28)
```

Kind of nitpicking as it is just a fallback but we don't want to target 28 yet. After changing it to 29 in the entire project we would have to adapt to some behavior changes: https://developer.android.com/about/versions/10/behavior-changes-10 which is probably not a good idea just before the release 😉